specs/core/1_custody-game.md

```diff
@@ -229,6 +230,18 @@ def epoch_to_custody_period(epoch: Epoch) -> int:
     return epoch // EPOCHS_PER_CUSTODY_PERIOD
 ```
 
+### `replace_empty_or_append`
+
+```python
+def replace_empty_or_append(list: List[Any], new_element: Any) -> int:
+    for i in range(len(list)):
+        if list[i] == empty(typeof(new_element)):
```

Comment:
`empty` refers to an SSZ object of the type filled with default values?
This is a bit tough to parse